fix: protobufjs/minimal cannot be resolved by @google-cloud/dlp

## Describe the bug

Next.js fails to resolve `protobufjs/minimal` when bundling `@google-cloud/dlp`. The error occurs because `@google-cloud/dlp` references `protobufjs/minimal` in its AMD branch (`protos.js:19`), but pnpm's strict dependency isolation doesn't link `protobufjs` as a direct dependency of `@google-cloud/dlp` — it's only available transitively through `google-gax`.

```
Module not found: Can't resolve 'protobufjs/minimal'

./node_modules/.pnpm/@google-cloud+dlp@6.5.0/node_modules/@google-cloud/dlp/build/protos/protos.js (19:9)
```

Import trace:

```
@google-cloud/dlp -> piiCheck.ts -> span-pii-redaction.service.ts -> recordSpanCommand.ts -> pipeline.ts -> pipelineRegistry.ts -> presets.ts -> instrumentation.ts
```

## To reproduce

1. Run `pnpm dev` (or `pnpm build`) in a fresh worktree after rebase onto latest main
2. Observe the `protobufjs/minimal` resolution error

## Expected behavior

`@google-cloud/dlp` should resolve `protobufjs/minimal` without issues.

## Root cause

`@google-cloud/dlp@6.5.0` has an implicit dependency on `protobufjs` (used in its AMD loader path in `protos.js`), but doesn't declare it in its own `package.json` — it relies on `google-gax` to bring it in. Under pnpm's strict isolation, this phantom dependency isn't available.

## Fix

Add `protobufjs` as a direct dependency in `langwatch/package.json`:

```json
"protobufjs": "^7.5.4"
```

Alternatively, use `pnpm-workspace.yaml` `packageExtensions` to inject it:

```yaml
packageExtensions:
  '@google-cloud/dlp':
    dependencies:
      protobufjs: '^7.5.4'
```

## Additional context

- `@google-cloud/dlp` was added by Rogerio in commit fd33a6a5e (March 2024)
- The issue may have been latent and only surfaced after a lockfile update changed hoisting behavior

--- a/langwatch/pnpm-workspace.yaml
+++ b/langwatch/pnpm-workspace.yaml
@@ -30,6 +30,9 @@ packageExtensions:
   bullmq-otel:
     dependencies:
       tslib: "^2.6.0"
+  "@google-cloud/dlp":
+    dependencies:
+      protobufjs: "^7.5.4"
 
 onlyBuiltDependencies:
   - '@prisma/client'

